Lib/zoneinfo/_zoneinfo.py

```diff
@@ -742,7 +742,7 @@ def _parse_tz_delta(tz_delta):
 
     if not -86400 < total < 86400:
         raise ValueError(
-            "Offset must be strictly between -24h and +24h:" + tz_delta
+            f"Offset must be strictly between -24h and +24h:{tz_delta}"
```

Comment:
```suggestion
            f"Offset must be strictly between -24h and +24h: {tz_delta}"
```

I was going to say this seems unnecessary, but then I realized that it's a lot easier to see the missing space with an f-string...